YIF_XUnitTests/Integration/YIF_Backend/Controllers/SuperAdminControllerTest.cs

```diff
@@ -370,5 +370,18 @@ public async Task GetIoEAdminIdByIoEId_EndpointReturnsOk()
             // Assert
             response.EnsureSuccessStatusCode();
         }
+
+        [Fact]
+        public async Task VerifyDeletedSpecialty() 
```

Comment:
meaningful method. rename to DeletedSpecialty_EndpointReturnsOk

Reply:
Done)